bootloader/grub.go

```diff
@@ -496,21 +496,69 @@ func staticCommandLineForGrubAssetEdition(asset string, edition uint) string {
 	return string(cmdline)
 }
 
+type taggedPath struct {
+	tag  string
+	path string
+}
+
+func (t taggedPath) GetId() string {
```

Comment:
nitpick: per go naming conventions this should just be Id

Reply:
Done.